client/state/comments/actions.js

```diff
@@ -122,7 +133,8 @@ export function requestPostComments( siteId, postId, status = 'approved' ) {
 		const requestId = createRequestId( siteId, postId, query );
 
 		// if the request status is in-flight or completed successfully, no need to re-fetch it
-		if ( postCommentRequests && [ COMMENTS_REQUEST, COMMENTS_REQUEST_SUCCESS ].indexOf( postCommentRequests.get( requestId ) ) !== -1 ) {
+		if ( postCommentRequests &&
+			[ COMMENTS_REQUEST, COMMENTS_REQUEST_SUCCESS ].indexOf( get( postCommentRequests, [ requestId ] ) ) !== -1 ) {
```

Comment:
don't really want `get` here. if postCommentRequests is null, we want this to explode.